summingbird-storm/src/main/scala/com/twitter/summingbird/storm/builder/BoltBuilder.scala

```diff
@@ -0,0 +1,38 @@
+package com.twitter.summingbird.storm.builder
+
+import com.twitter.summingbird.option.JobId
+import org.apache.storm.topology.IRichBolt
+import org.apache.storm.tuple.Fields
+
+private[builder] object BoltBuilder {
```

Comment:
Curios why you've built this wrapper by moving code out of BaseBolt. Is the intention to remove knowledge of Topology and EdgeTypes from BaseBolt? If so then may be add a comment to BaseBolt to state this design goal.